osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/PlayerArea.cs

```diff
@@ -93,7 +98,13 @@ public void LoadScore([NotNull] Score score)
                 }
             };
 
-            stack.Push(new MultiSpectatorPlayerLoader(Score, () => new MultiSpectatorPlayer(Score, GameplayClock)));
+            stack.Push(new MultiSpectatorPlayerLoader(Score, () =>
+            {
+                var player = new MultiSpectatorPlayer(Score, GameplayClock);
+                player.OnGameplayStarted += OnGameplayStarted;
```

Comment:
I believe this needs to be `+= () => OnGameplayStarted?.Invoke(value)` to be safe.